wiki/Help_centre/Upgrading_to_lazer/vi.md

```diff
@@ -0,0 +1,288 @@
+
+<!-- lint ignore no-heading-punctuation -->
+
+#### Tôi thích màn hình ghi điểm cổ điển, lúc mà điểm số trở nên thực sự lớn.
+
+Bạn thực sự có thể thay đổi cài đặt `Chế độ hiển thị điểm` thành `Cổ điển` để lấy lại phong cách ghi điểm trong toàn trò chơi! Nó sẽ không phải là một sự kết hợp hoàn hảo, nhưng sẽ mang lại cho bạn cảm giác cổ điển giống như vậy và được áp dụng ở mọi nơi mà bạn mong đợi.
+
+#### Đặt điểm trên lazer thì nó có tồn tại vĩnh viễn không?
+
+Mặc dù chúng tôi sẽ cố gắng duy trì càng nhiều điểm càng tốt, nhưng chúng tôi **không đảm bảo rằng điểm sẽ duy trì vô thời hạn**. Tại bất kỳ thời điểm nào, chúng tôi có thể chọn xóa một tập hợp con hoặc tất cả các điểm để duy trì sự cân bằng của trò chơi.
+
+#### Điểm số được đặt ở chế độ ổn định cuối cùng sẽ hiển thị ở lazer?
+
+Đúng. Sau khi chúng tôi hoàn tất việc cân bằng sự kết hợp giữa lazer và điểm số ổn định, cả hai sẽ hiển thị.
+
+#### Tất cả các mod sẽ được xếp hạng?
+
+Hiện tại, điểm số của tất cả các kết hợp mod xuất hiện trên bảng xếp hạng. Liệu điểm số có cho điểm hiệu suất với tất cả các mod hay không (và nếu có, liệu sẽ có phần thưởng hay hình phạt được áp dụng hay không) vẫn còn đang được thảo luận.
+
+#### Tôi không thích cơ chế chơi trò chơi mới. Tôi có thể khôi phục cơ chế chơi trò chơi cũ như ở phiên bản ổn định không?
+
+Vui lòng thử mod "cổ điển", mod này sẽ khôi phục phần lớn hành vi cũ mà bạn đã quen. Ngoài ra, hãy đảm bảo kiểm tra các cài đặt do mod cổ điển cung cấp, vì nó sẽ cho phép bạn tùy chỉnh thêm trải nghiệm của mình và cũng hiểu những thay đổi nào đang được áp dụng (vì tất cả chúng đều được liệt kê ở đó).
+
+### Skinning và giao diện người dùng
+
+#### Một cái gì đó đang hoạt động khác với ổn định và tôi không thích nó!
+
+Vui lòng chạy trình hướng dẫn thiết lập ở đầu cài đặt và xem qua các cài đặt trên màn hình `Hành vi`. Rất nhiều cài đặt phổ biến đã thay đổi giá trị mặc định được liệt kê ở đây. Ngoài ra còn có một nút duy nhất mà bạn có thể nhấn để áp dụng các hành vi cũ làm điểm khởi đầu cho hành trình lazer của mình.
+
+#### Giao diện cũ cuối cùng có hoạt động trong màn hình chọn bài hát và kết quả không?
+
+Chúng tôi sẽ cố gắng hết sức để khôi phục lại nhiều thứ nhất có thể mà không chặn chức năng mới. Điều này sẽ đến sau này.
+
+#### Tôi có thể sử dụng con trỏ skin của mình trong menu không?
+
+Chúng tôi có thể sẽ mang lại hỗ trợ cho điều này trong tương lai do nhu cầu phổ biến.
+
+### Hiệu suất
+
+#### Tại sao tôi không thể sử dụng với không giới hạn FPS?
+
+Trên một ngưỡng nhất định, không có lý do gì để chạy ở tốc độ khung hình cao hơn. Lazer sử dụng nhiều công nghệ mới khác nhau để đảm bảo có thể đạt được độ trễ thấp nhất mà không yêu cầu tốc độ khung hình cao. Điều này sẽ tiếp tục cải thiện trong tương lai vì chúng tôi vẫn còn một số cải tiến cần thực hiện.
+
+Các cuộc thăm dò của Lazer cho đầu vào ở 1000 Hz bất kể bộ giới hạn FPS, đó là lý do tại sao cài đặt bộ giới hạn tối đa cũng sẽ giới hạn ở 1000 FPS.
```

Comment:
```suggestion
Tần suất poll input của Lazer được đặt ở 1000 Hz bất kể mức giới hạn FPS, đó là lý do tại sao thiết lập mức giới hạn tối đa cũng sẽ giới hạn ở 1000 FPS.
```
Leave terminologies untranslated. You can change the format though. "Lazer polls for input" -> "Input polling rate of Lazer" -> "Tần suất poll input của lazer"